README.md

```diff
@@ -50,7 +50,8 @@ See the [security documentation](https://docs.openshift.org/latest/admin_guide/m
 
 Getting Started
 ---------------
-The easiest way to run OpenShift Origin is in a Docker container (OpenShift requires Docker 1.6.2 or higher):
+The easiest way to run OpenShift Origin is in a Docker container (OpenShift requires Docker 1.6.2 or higher).
+If you'd like to get started with Atomic Enterprise please refer to the [quickstart guide](https://github.com/openshift/origin/blob/master/docs/atomic-quick-start-tutorial.adoc).
```

Comment:
Will that actually work when it's displayed on the main page with the `blob/{branch}` section?  Seems like it would be fine if you were already viewing it directly so the url was https://github.com/openshift/origin/blob/master/README.md and the relative link was `docs/quickstart.adoc` but would fail when the url is https://github.com/openshift/origin.

I can give it a test in a dummy repo and see.